src/control/directive.ts

```diff
@@ -112,7 +112,7 @@ export class NgrxFormControlDirective<TStateValue extends FormControlValueTypes,
     // for the dom parameter the `null` type must be last to ensure that in the compiled output
     // there is no reference to the Document type to support non-browser platforms
     @Optional() @Inject(DOCUMENT) private dom: Document | null,
-    private actionsSubject: ActionsSubject,
+    @Optional() private actionsSubject: ActionsSubject,
```

Comment:
if it is optional, then you should type it as `ActionsSubject | null` since Angular provides `null` for non-existing optional dependencies. This will ensure you are not accidentally accessing the subject anywhere unsafely

Reply:
That is not possible due to how the Angular DI mechanism works. If you extend it to union type `ActionsSubject | null` it will complain that it can not find the required `InjectionToken`. I have introduced a private field with the proper type declaration and just set it the parameter to the field.

Reply:
Hmm, I believe it would work if you specify `@Inject(ActionsSubject)` but I am honestly not 100% sure. I'll look into this myself after merging.

Reply:
Ah, I see, good to know that it can be done more elegantly with this.